tests/datasource/fluent/integration/test_sql_datasources.py

```diff
@@ -154,12 +157,8 @@ def _get_exception_details(
     ]
 ]:
     """Extract a list of exception_info dicts from a CheckpointResult."""
-    validation_results: list[
```

Comment:
This piece was erroring, so I figured it was a good opportunity to clean it up. Summary of changes:
* Doesn't convert the model to a dict
* Looks like there's no longer a `validation_result` object with nested `results`. Just a `results` key instead.